packages/diagrams/frontend/sirius-components-diagrams/src/converter/RectangleNodeConverter.ts

```diff
@@ -83,7 +85,9 @@ const toRectangularNode = (
     defaultWidth: gqlNode.defaultWidth,
     defaultHeight: gqlNode.defaultHeight,
     isBorderNode: isBorderNode,
-    borderNodePosition: isBorderNode ? BorderNodePosition.EAST : null,
+    borderNodePosition: isBorderNode
+      ? convertBorderNodePosition(gqlNode.initialBorderNodePosition, BorderNodePosition.EAST)
```

Comment:
Why should rectangle be in the east and others in the west? Do not hardcode specific default like that.